lib/samples/show_wfs_layer_from_url/show_wfs_layer_from_url.dart

```diff
@@ -0,0 +1,152 @@
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/utils/sample_state_support.dart';
+import 'package:flutter/material.dart';
+
+class ShowWfsLayerFromUrl extends StatefulWidget {
+  const ShowWfsLayerFromUrl({super.key});
+
+  @override
+  State<ShowWfsLayerFromUrl> createState() => _ShowWfsLayerFromUrlState();
+}
+
+class _ShowWfsLayerFromUrlState extends State<ShowWfsLayerFromUrl>
+    with SampleStateSupport {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+
+  // Reference to the WFS feature table.
+  late WfsFeatureTable _featureTable;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: false,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                  ),
+                ),
+              ],
+            ),
+            // Display a progress indicator and prevent interaction until state is ready.
+            Visibility(
+              visible: !_ready,
+              child: const SizedBox.expand(
+                child: ColoredBox(
+                  color: Colors.white30,
+                  child: Center(child: CircularProgressIndicator()),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  void onMapViewReady() async {
+    // Create a map with the ArcGIS Navigation basemap style and set to the map view.
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISNavigation);
+
+    _mapViewController.arcGISMap = map;
+
+    // Load the WFS layer.
+    await loadWfsLayerFromURL();
+
+    // Set the initial viewpoint to Seattle Downtown using latitude, longitude and scale.
+    final initialViewPoint = Viewpoint.withLatLongScale(
+      latitude: 47.617207,
+      longitude: -122.341581,
+      scale: 5000,
+    );
+
+    _mapViewController.setViewpoint(initialViewPoint);
+
+    // Add a listener for map navigation events
+    _mapViewController.onNavigationChanged.listen((isNavigating) {
+      if (!isNavigating) {
+        loadFeatures();
+      }
+    });
+
+    await loadFeatures();
```

Comment:
You may want to add a comment for this line saying that this is needed to load the features in the initial viewpoint

Reply:
Done added the comment for `loadFeatures()`